app/views/rubric_criteria/_rubric_criterion_level.html.erb

```diff
@@ -1,25 +1,25 @@
 <%#
 # Erb template for display rubric criterion levels in the assignment rubric manager
 %>
-<div class="rubric_level_header">
-  <%= f.label "level_#{rubric_level_index}_name".to_sym,
-              I18n.t("rubric_criteria.level.level_index",
-                     index: rubric_level_index),
-              class: "bold_inline_label" %>
-</div>
-<div>
-  <%= f.label :name, Level.human_attribute_name(:name), class: 'required' %>
-  <%= f.text_field :name, value: "#{level.name}"%>
-</div>
-<div>
-  <%= f.label :mark, Level.human_attribute_name(:mark) + " (#{level.mark}/#{criterion.max_mark})",
-              class: 'required' %>
-  <%= f.number_field :mark, value: "#{level.mark}", step: 0.1%>
-</div>
-<div>
-  <%= f.label :description, Level.human_attribute_name(:description), class: 'description required' %>
-  <%= f.text_area :description, value: "#{level.description}"%>
-</div>
+<% i=0 %>
```

Comment:
This use of an index is okay. In terms of style, always put spaces around `=`, and also put spaces after the `<%`/`<%=` and before the `%>` (you need to fix this throughout this file).